addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxThingHandler.java

```diff
@@ -61,23 +65,47 @@
     private FreeboxHandler bridgeHandler;
     private Calendar lastPhoneCheck;
     private String netAddress;
+    private String airPlayName;
+    private String airPlayPassword;
 
     public FreeboxThingHandler(Thing thing) {
         super(thing);
-
-        phoneJob = null;
-        callsJob = null;
-        bridgeHandler = null;
-        netAddress = null;
     }
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command == null || command instanceof RefreshType) {
```

Comment:
I am not sure but I was thinking that we can receive REFRESH commands before the thing is ONLINE. Is it true ?
But as I don't handle REFRESH commands, the result will be the same with your proposal.